src/App.tsx

```diff
@@ -1,57 +1,27 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import './App.scss';
+import { AutoComplete } from './components/AutoComplete/AutoComplete';
```

Comment:
```suggestion
import { Autocomplete } from './components/Autocomplete/Autocomplete';
```
Make sure that the case path matches the file to avoid errors